src/components/HtmlOutput/index.tsx

```diff
@@ -40,21 +41,29 @@ function useSanitizedHtml(rawHtml: string | null | undefined) {
                     p: ['style'],
                     span: ['style'],
                     div: ['style'],
-                    img: ['src', 'width', 'height', 'style', 'alt'],
-                    iframe: ['src', 'width', 'height', 'frameborder', 'style'],
-                    a: ['href'],
+                    img: ['style', 'src', 'width', 'height', 'alt'],
+                    iframe: ['style', 'src', 'width', 'height', 'frameborder'],
+                    a: ['style', 'href'],
                 },
                 allowedSchemes: ['http', 'https', 'data'],
+                /*
                 allowedStyles: {
                     '*': {
-                        // Allow indentation
-                        'padding-left': [/^\d+(?:px)$/],
-                        'font-size': [/^\d+(?:px)$/],
+                        'padding-left': [/^\d+px$/],
+                        'font-size': [/^\d+px$/],
                         'text-align': [/.+/],
                     },
+                    iframe: {
+                        width: [/^\d+px$/],
+                        height: [/^\d+:px$/],
+                        border: [/.+/],
+                    },
                 },
+                */
```

Comment:
This will make the sanitization much weaker, let's try to limit numbers of allowed styles.